tests/test0000.d

```diff
@@ -1,3 +1,5 @@
+// RUN: %sdc %s -o %t && sh -c '%t; [ $? = 42 ]'
```

Comment:
Well, @UplinkCoder had a good first shot at it, but never finished it. I still think that it was the right way forward and hope this can be nailed at some point.

Also I sent you a slack invite so we can talk on a more direct way.